common/src/main/java/org/conscrypt/NativeCrypto.java

```diff
@@ -52,13 +68,39 @@
 @Internal
 public final class NativeCrypto {
     // --- OpenSSL library initialization --------------------------------------
+    private static final Throwable UNAVAILABILITY_CAUSE;
     static {
-        NativeCryptoJni.init();
-        clinit();
+        Throwable cause = null;
```

Comment:
No, and it seems like an absolute priority for a lot of usecases for something like Conscrypt, as an optimisation over the built in JVM TLS, that you can check Conscrypt.isAvailable() without having the code blow up first as part of incidentally hitting some static code.